storm-client/src/jvm/org/apache/storm/security/auth/workertoken/WorkerTokenAuthorizer.java

```diff
@@ -41,7 +41,7 @@
 /**
  * Allow for SASL authentication using worker tokens.
  */
-public class WorkerTokenAuthorizer implements PasswordProvider, Closeable {
+public class WorkerTokenAuthorizer implements PasswordProvider, AutoCloseable, Closeable {
```

Comment:
`Closeable` extends `AutoCloseable`, so you don't need to make a change.

Reply:
You're right, fixed.